shared/macros.jinja

```diff
@@ -540,3 +540,17 @@ ocil_clause: "the correct value is not returned"
     {{%- endif %}}
 {{%- endmacro %}}
 
+
+{{# Call e.g. {{{ if_product_name_is(rhel6="Old RHEL", rhel5="Very old RHEL") }}} #}}
+{{% macro if_product_name_is() -%}}
+    {{% if product in kwargs -%}}
+	{{{ kwargs[product] }}}
+    {{%- endif %}}
+{{%- endmacro %}}
+
+
+{{% macro if_product_pkg_system_is() -%}}
```

Comment:
same here...

Reply:
Converting this to a `if/elif` statement is much more feasible, but still, this is code where swich/case statement would be the best, and in Python => jinja, using dictionaries instead is a [widely accepted practice](https://stackoverflow.com/questions/60208/replacements-for-switch-statement-in-python). `kwargs` is such dictionary.